tests/fs-test.rs

```diff
@@ -37,24 +62,23 @@ fn uhyvefilemap_test() {
 	let bin_path = build_hermit_bin("create_file");
 
 	let mut params = Params {
-		cpu_count: 2.try_into().unwrap(),
+		cpu_count: 1.try_into().unwrap(),
 		memory_size: Byte::from_u64_with_unit(32, Unit::MiB)
 			.unwrap()
 			.try_into()
 			.unwrap(),
-		file_mapping: vec!["foo.txt:wrong.txt".to_string()],
+		file_mapping: vec!["./foo.txt:/root/dir/wrong.txt".to_string()],
```

Comment:
We will have to port this PR's changes to the tests as soon as the new kernel release is available, because the new kernel version will use `/root/`/absolute paths.